power-control-x86/src/power_control.cpp

```diff
@@ -1989,6 +1994,21 @@ int main(int argc, char* argv[])
         return -1;
     }
 
+    // check RSMRST line for max 1 sec
+    gpioLine = gpiod::find_line(MonRSMRST);
+    if(gpioLine)
+    {
+        for (int i=0; i < MON_RSMRST_MAX_LOOP; i++)
```

Comment:
This loop logic is inconsistent with the existing code. Either wait for a second before checking for MonRSMRST or use interrupt driven mechanism to state check,

Reply:
done. add 1 sec delay in case RSMRST is not set yet.